### ISSUE TITLE
[PERFORMANCE]: Progress Button - CONSIDER importing Lodash Unique ID only

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10439]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [Progress Button component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/progressbutton/) imports the entire Lodash library as `_` which seems like it could add a fair bit of weight to the component. Could we consider changing this `import` to use just the Lodash unique ID function?

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Screen readers ignore the visual text flourishes like double angle brackets before or after button text
- [ ] No axe violations

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/progressbutton/

## Possible Fixes (optional)

```diff
ProgressButton.jsx#L3
import PropTypes from 'prop-types';
import React from 'react';
- import _ from 'lodash';
+ import { uniqueId} from 'lodash';
...
ProgressButton.jsx#L11
- this.id = _.uniqueId();
+ this.id = uniqueId();
```

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #219](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/219) via [**ZenHub**](https://www.zenhub.com/)
